.github/workflows/ci.yml

```diff
@@ -118,3 +118,19 @@ jobs:
           PGP_SECRET: ${{ secrets.PGP_SECRET }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+      
+      - name: Compile Documentation Markdown Files
+        if: startsWith(github.ref, 'refs/tags/v') # do not run for snapshots
+        run: sbt "docs/mdoc"
```

Comment:
Not positive this is needed here since we run it above in the CI steps. I am just not sure if the results will be cached along with the other compilation artifacts

Reply:
Seems worth checking, if we can avoid running sbt I'd give it a try :)